composer/algorithms/cutmix/cutmix.py

```diff
@@ -200,13 +202,19 @@ class CutMix(Algorithm):
             )
     """
 
-    def __init__(self, num_classes: int, alpha: float = 1., uniform_sampling: bool = False):
+    def __init__(self,
+                 num_classes: int,
+                 alpha: float = 1.,
+                 uniform_sampling: bool = False,
+                 input_key: Any = 0,
```

Comment:
Can this type be more specific? as a clue to the user?

Reply:
I think in the past we decided on `Any` as mentioned [here](https://github.com/mosaicml/composer/pull/978#discussion_r862994938)

Reply:
Hmm, we _could_ do a `Union[str, int, callable, Any]`, but that's just more verbose than Any for how pyright will interpret it :)
Perhaps could we clarify in the docstring what this can be, and recommendations for common batch formats?

Reply:
I'm fine either way, just that the `Any` typing hint well.. doesn't provide much of a hint to users.

Reply:
ok changed to `Union[str, int, callable, Any]` here: [a95b28c](https://github.com/mosaicml/composer/pull/1041/commits/a95b28c0b12f769c315eec6a6c4ae5b62369f076)